lincheck/src/test/java/org/jetbrains/kotlinx/lincheck/test/verifier/CustomScenarioDSL.kt

```diff
@@ -64,51 +64,49 @@ fun verify(
     if (expected) assert(res) else assert(!res)
 }
 
+fun actor(f: KFunction<*>, vararg args: Any?): Actor =
```

Comment:
Get rid of this delegation - use `actor` function directly

Reply:
Is it ok that then `actor` should be imported separately from this execution DSL?